docs/installing.rst

```diff
@@ -148,8 +148,22 @@ The following virtualization providers are suggested for the respective operatin
     # Set libvirtd to start automatically on system boot
     $ sudo systemctl enable libvirtd
 
+---------------------------------------------------
+3. Install dependencies for vagrant-service-manager
```

Comment:
I'd not list the dependencies per plugin. I'd rather list everything one needs to install on Fedroa to get going in one go.

And  I am against listing all these detailed instructions on vagrant-serice-manager. The service-manager docs already point to ADB docs for more detailed help. I think we can keep it this way.